src/state/reducers/resource.ts

```diff
@@ -33,10 +37,11 @@ export type ResourceState = {
 const initialState: ResourceState = {
   resources: [],
   structureDefinition: null,
+  extensionStructureDefinition: (extensionStructureDefinition as unknown) as StructureDefinition.AsObject,
```

Comment:
Does typescript return an error if you try to cast directly 'extensionStructureDefinition' like so :
```suggestion
  extensionStructureDefinition: extensionStructureDefinition as StructureDefinition.AsObject,
```

Reply:
![image](https://user-images.githubusercontent.com/48218773/91732368-6e8d2b80-eba8-11ea-8ec3-a7d611d569be.png)
We got this error